### ISSUE TITLE
Enclosure: Reset button

### ISSUE THREAD

**Issue by chipaudette** [Tympan/Tympan_Earpiece_Shield#3]
@eyuan-creare, In a future revision of the enclosure, we need to add a hole so that we can poke down and hit the Tympan's reset button.

I've got a Tympan + Earpiece Shield that won't reprogram.  It looks like I have to open it up and press the reset button.

**Comment by eyuan-creare** [comment]
Good idea.  For the existing units, I can drill out a small hole if you think Marc and Daniel would care.  For future units, I can add a flex section to the housing to create a button that rests on the reset.

-Eric
